cypress/integration/dynamic_sim_test_spec.js

```diff
@@ -11,32 +15,36 @@ const modelSettings = new ModelSettings;
 context('Dynamic Diagram sim setting tests', function(){
     describe('Set up a static basic diagram', function(){
         it('will set up document as a static basic ', function() {
-            cy.getSageIframe().find(document.simSettingsToolButton()).click();
-            cy.getSageIframe().find(simSettings.settingRadioStatic()).contains("Dynamic").siblings('input[type="radio"]').check();
-            cy.getSageIframe().find(simSettings.settingSubmenu()).contains("Basic").siblings('input[type="radio"]').check();
+            SageModelerHelper.updateSimulationModelSettings('dynamic');
```

Comment:
I would change this test to set up a dynamic basic model since you are in the dynamic_sim_test.